src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -150,20 +170,18 @@ public static class TextLoaderSaverCatalog
         /// <param name="catalog">The <see cref="DataOperationsCatalog"/> catalog.</param>
         /// <param name="path">The path to the file.</param>
         /// <param name="separatorChar">Column separator character. Default is '\t'</param>
-        /// <param name="hasHeader">Whether the file has a header with feature names. Note: If a TextLoader is created with hasHeader = true but without a
-        /// dataSample, then vector columns made by TextLoader will not contain slot name annotations (slots being the elements of the given vector column),
-        /// because the output schema is made when the TextLoader is made, and not when <see cref="TextLoader.Load(IMultiStreamSource)"/> is called.
-        /// In addition, the case where dataSample = null and hasHeader = true indicates to the loader that when it is given a file when Load()
-        /// is called, it needs to skip the first line.</param>
-        /// <param name="allowQuoting">Whether the input may include quoted values,
-        /// which can contain separator characters, colons,
-        /// and distinguish empty values from missing values. When true, consecutive separators
-        /// denote a missing value and an empty value is denoted by \"\".
-        /// When false, consecutive separators denote an empty value.</param>
-        /// <param name="trimWhitespace">Remove trailing whitespace from lines</param>
-        /// <param name="allowSparse">Whether the input may include sparse representations for example,
-        /// if one of the row contains "5 2:6 4:3" that's mean there are 5 columns all zero
-        /// except for 3rd and 5th columns which have values 6 and 3</param>
+        /// <param name="hasHeader">Whether the file has a header with feature names. When <see langword="true"/>, the loader will skip the first line when
```

Comment:
>feature names [](start = 71, length = 13)

Since this overload won't create feature names (because it doesn't pass the data sample to the constructor), I would not mention it, and just leave the second sentence.